Source/Common/Include/MPIWrapper.h

```diff
@@ -376,6 +376,20 @@ class MPIWrapper : public std::enable_shared_from_this<MPIWrapper>
     {
         return m_myRank;
     }
+
+    size_t CurrentLocalNodeRank()
+    {
+        const char* p = std::getenv("OMPI_COMM_WORLD_LOCAL_RANK");
```

Comment:
I stated this will not work with windows and asked amit about the windows alternative of this. Seems there is no such implementation as far as we know in msmpi.